packages/fxa-auth-server/bin/key_server.js

```diff
@@ -224,7 +226,7 @@ async function run(config) {
   }
 
   const twilio = TwilioFactory.useFactory(config.twilio);
-  const recoveryPhoneRedis = require('../lib/redis')({
+  const recoveryPhoneRedis = new Redis({
```

Comment:
@vpomerleau This was the source of the issue. FxARedis didn't expose the `scan` function. I'm glad we caught this today rather than later. All the tests in the recovery phone lib are designed to work with the regular Redis client, and while the FxARedis client is basically just a wrapper that mostly worked, I'd much prefer using the same implementation the lib is put under test with. It’s kind of sad we don’t have type checking in this file, cause that would have also caught this oversight much sooner.